build

```diff
@@ -2,11 +2,22 @@
 set -e
 export LC_ALL=C
 
+if [ "$1" = "meson" ]; then
+    use_meson="$1"; shift
+fi
+
 #scripts/fribidi-config
 #scripts/fribidi-build "$@"
 scripts/libass-config
 scripts/libass-build "$@"
 scripts/ffmpeg-config
 scripts/ffmpeg-build "$@"
-scripts/mpv-config
-scripts/mpv-build "$@"
+
+# waf throws an error if you pass an empty ""
+if [ -n "$use_meson" ]; then
+    scripts/mpv-config "$use_meson"
+    scripts/mpv-build "$use_meson" "$@"
+else
+    scripts/mpv-config
+    scripts/mpv-build "$@"
+fi
```

Comment:
You can probably get away with
```
scripts/mpv-config $use_meson
scripts/mpv-build $use_meson "$@"
```
here

Reply:
Unfortunately, Waf complains with a `No function '' defined in /home/alec/src/mpv-build/mpv/wscript` and returns an error if you pass it an "".